easybuild/easyconfigs/d/dftd4/dftd4-3.4.0-iimkl-2022a.eb

```diff
@@ -0,0 +1,43 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'dftd4'
+version = '3.4.0'
+
+homepage = 'https://github.com/dftd4/dftd4'
+description = """Generally Applicable Atomic-Charge Dependent London Dispersion Correction."""
+
+toolchain = {'name': 'iimkl', 'version': '2022a'}
+
+github_account = 'dftd4'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['f3b0a16a952817ae48e819626e13676fba3b61c8beea47b0f8ada2fbb679fb7b']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.10.2'),
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+    ('multicharge', '0.2.0'),
+]
+
+configopts = '-DBLAS_LIBRARIES="-lmkl_sequential -lmkl_intel_lp64 -lmkl_core" '
+configopts += '-DLAPACK_LIBRARIES="-lmkl_sequential -lmkl_intel_lp64 -lmkl_core" '
```

Comment:
should work with BLAS_LIBRARIES="$LIBBLAS" and LIBLAPACK respectively

Reply:
Good idea, thanks!